pulp_rpm/tests/functional/constants.py

```diff
@@ -329,6 +329,14 @@
     RPM_KICKSTART_CONTENT_NAME: RPM_KICKSTART_COUNT
 }
 
+RPM_KICKSTART_REPOSITORY_DIRECTORIES = [
+    '.treeinfo',
```

Comment:
A nit-pick. '.treeinfo' is a file, no a directory.
I'm not sure what's the better name though. Maybe RPM_KICKSTART_REPO_ROOT_CONTENT? 

Not sure. Up to you.

Reply:
I agree that 'root content' in a variable name is more descriptive.